### PR TITLE
Kubecon related changes

### PR DESCRIPTION
**Pull request by sriumcp**

### CODE REVIEW

File: docs/getting-started/concepts.md [hunk 1]
```diff
@@ -13,54 +13,33 @@ Iter8 supports the following use-cases.
 
 
 ## Iter8 experiment
-Iter8 introduces the notion of an experiment, which is a list of configurable tasks that are executed in a specific sequence.
+The main notion introduced by Iter8 is that of an experiment. An experiment is simply a list of tasks that are executed in a specific sequence.
```
Comment (Alan-Cha) [inline docs/getting-started/concepts.md:16]: I think I would prefer

```suggestion
Iter8 introduces the notion of an "experiment". An experiment is simply a list of tasks that are executed in a specific sequence.
```

File: docs/getting-started/concepts.md [hunk 2]
```diff
@@ -13,54 +13,33 @@ Iter8 supports the following use-cases.
...
 
 <p align='center'>
-  <img alt-text="load-test-http" src="../../images/iter8-intro-dark.png" width="70%" />
+  <img alt-text="Iter8 experiment" src="../../images/authoring.png" width="40%" />
 </p>
 
-Iter8 packs a number of powerful features that facilitate Kubernetes app testing and experimentation. They include the following.
+Iter8 defines a variety of configurable tasks. Authoring an experiment is as simple as specifying the names of the tasks you wish to include in the experiment, and specifying their parameter values. The following are some examples of tasks provided by Iter8.
```
Comment (Alan-Cha) [inline docs/getting-started/concepts.md:22]:
```suggestion
Iter8 provides a variety of configurable tasks. Authoring an experiment is as simple as specifying the names of the tasks and specifying their parameter values. The following are some examples of tasks provided by Iter8.
```

File: docs/getting-started/your-first-experiment.md [hunk 3]
```diff
@@ -81,15 +81,20 @@ iter8 k log
 ***
 
 ## Cleanup
-Remove the [Kubernetes objects](https://kubernetes.io/docs/concepts/overview/working-with-objects/kubernetes-objects/) created by the `iter8 k launch` command.
+Remove the Iter8 experiment and the sample  app from the Kubernetes cluster, and the local Iter8 `charts` folder.
```
Comment (Alan-Cha) [inline docs/getting-started/your-first-experiment.md:84]:
```suggestion
Remove the Iter8 experiment and the sample app from the Kubernetes cluster and the local Iter8 `charts` folder.
```

File: docs/tutorials/load-test-grpc.md [hunk 4]
```diff
@@ -58,4 +58,15 @@ iter8 k launch \
...
 ***
 
-Assert experiment outcomes, view experiment report, view experiment logs, and cleanup as described in [your first experiment](../getting-started/your-first-experiment.md).
\ No newline at end of file
+Assert experiment outcomes, view experiment report, view experiment logs, and cleanup as described in [your first experiment](../getting-started/your-first-experiment.md).
+
+***
+
+## Cleanup
+Remove the Iter8 experiment and the sample  app from the Kubernetes cluster, and the local Iter8 `charts` folder.
```
Comment (Alan-Cha) [inline docs/tutorials/load-test-grpc.md:66]:
```suggestion
Remove the Iter8 experiment and the sample app from the Kubernetes cluster and the local Iter8 `charts` folder.
```

**Review by kalantar — DISMISSED** [review]

File: docs/getting-started/concepts.md (hunk 1 above)
Comment (sriumcp) [reply docs/getting-started/concepts.md]: Fixed.

File: docs/getting-started/concepts.md (hunk 2 above)
Comment (sriumcp) [reply docs/getting-started/concepts.md:22]: Fixed.

File: docs/getting-started/your-first-experiment.md (hunk 3 above)
Comment (sriumcp) [reply docs/getting-started/your-first-experiment.md:84]: Fixed.

File: docs/tutorials/load-test-grpc.md (hunk 4 above)
Comment (sriumcp) [reply docs/tutorials/load-test-grpc.md]: Fixed.

**Pull request merged**